c2corg_ui/static/js/map/map.js

```diff
@@ -636,11 +636,23 @@ app.MapController.prototype.handleEditModelChange_ = function(event, data) {
 
 
 /**
+ * @param {ol.Feature} feature
+ * @return {boolean}
+ * @private
+ */
+app.MapController.prototype.isLineFeature_ = function(feature) {
```

Comment:
Do you maybe want to move this function into `app.utils`? It's the same as you used for the download?